server/libs/database/src/migrations/1661270339373-AddAssetChecksum.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddAssetChecksum1661270339373 implements MigrationInterface {
+    name = 'AddAssetChecksum1661270339373'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "assets" ADD "checksum" character varying(40)`);
```

Comment:
Is checksum value always limited to 40 characters?

Reply:
SHA1 just have 40 bytes in length.

Reply:
Other alg may have different length. For optimization, I use 40, how about you?

Reply:
I am fine with 40 chars, just wondering since I haven't done this before.

Reply:
I guess we don't need to send the hashes to any client. Even if we do, I'd still rather do the conversion and store the hash more efficiently in memory & DB

Reply:
@zoodyy  We don't need return hash to client app, but we need to search/filter internally, so can we search with `bytea` easily?

Reply:
In postgres, we can index the hash column also if it's `bytea` and search efficiency